caffe2/python/pybind_state.cc

```diff
@@ -229,6 +234,12 @@ class GetPythonGradient : public GradientMakerBase {
   }
 };
 
+#ifdef CAFFE2_USE_IDEEP
+REGISTER_IDEEP_OPERATOR(
+   Python,
+   IDEEPFallbackOp<PythonOp<CPUContext, false>>);
+#endif
+
```

Comment:
Better to place this fallback declaration in pybind_state_ideep.cc?

Reply:
fixed